src/IO/WriteBufferFromS3TaskTracker.h

```diff
@@ -22,7 +20,7 @@ namespace DB
 /// Basic exception safety is provided. If exception occurred the object has to be destroyed.
 /// No thread safety is provided. Use this object with no concurrency.
 
-class WriteBufferFromS3::TaskTracker
+class TaskTracker
```

Comment:
Let's rename it to `ThreadPoolTaskTracker` and move to `Common/`.

Reply:
addressed it